src/v2/Apps/Home/Components/HomePersonalizeMore/HomePersonalizeMore.story.tsx

```diff
@@ -0,0 +1,14 @@
+import { HomePersonalizeMore } from "./HomePersonalizeMore"
+import { States } from "storybook-states"
+
+export default {
+  title: "Components/HomePersonalizeMore",
+}
+
+export const Default = () => {
+  return (
+    <States>
+      <HomePersonalizeMore />
```

Comment:
So you did this just in storybooks and then when we settle the questions we can get this onto the homepage, yeah?